engine/CMakeLists.txt

```diff
@@ -1092,7 +1128,11 @@ SET(TST_LIBS vegastrike-engine_com vegastrike-OPcollide ${TST_LIBS})
 INCLUDE_DIRECTORIES(${TST_INCLUDES})
 
 TARGET_LINK_LIBRARIES(vegastrike ${TST_LIBS})
-SET_TARGET_PROPERTIES(vegastrike PROPERTIES LINK_FLAGS "-L/usr/lib ${TST_LFLAGS}")
+IF (MSVC)
+    SET_TARGET_PROPERTIES(vegastrike PROPERTIES LINK_FLAGS "${TST_LFLAGS}")
+ELSE (MSVC)
+    SET_TARGET_PROPERTIES(vegastrike PROPERTIES LINK_FLAGS "-L/usr/lib ${TST_LFLAGS}")
+endif (MSVC)
```

Comment:
All cmake directives should be uppercase

Reply:
yeah, that's me fighting the editor when modifying the cmake file in Visual Studio 2019 (I should just use emacs instead :-P ) - it wants to "autocorrect" every cmake directive to lowercase -- :it is a mystery: -- and I must have missed that one :-|

Pushing the1-liner fix for this; assuming that the checks all re-run successfully, I'll do the merge once they complete.

Reply:
@ministerofinformation I know there's a few others; but either way - feel free to merge once it passes the PR gates since it's just build tooling changes.